includes/reader-revenue/class-woocommerce-connection.php

```diff
@@ -186,7 +188,11 @@ public static function get_contact_from_order( $order ) {
 
 		$metadata[ Newspack_Newsletters::get_metadata_key( 'account' ) ]           = $order->get_customer_id();
 		$metadata[ Newspack_Newsletters::get_metadata_key( 'registration_date' ) ] = $customer->get_date_created()->date( Newspack_Newsletters::METADATA_DATE_FORMAT );
-		$metadata[ Newspack_Newsletters::get_metadata_key( 'payment_page' ) ]      = \wc_get_checkout_url();
```

Comment:
Note that `payment_page` meta field value [is computed](https://github.com/Automattic/newspack-plugin/blob/173121e41d23bfe6fb1044d3b84bdfe5860ccac9/includes/plugins/class-newspack-newsletters.php#L235-L244) in `Newspack_Newsletters_Subscription::add_contact`, it's not meant to be passed directly. So it's removed here.